pkg/apis/componentconfig/types_pluginargs.go

```diff
@@ -131,3 +133,19 @@ type HighNodeUtilizationArgs struct {
 	Thresholds    api.ResourceThresholds
 	NumberOfNodes int
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// DefaultEvictorArgs holds arguments used to configure DefaultEvictorArgs plugin.
+type DefaultEvictorArgs struct {
```

Comment:
Would you please move `DefaultEvictorArgs` with the deep copy generator under `pkg/framework/plugins/defaultevictor`? E.g. `pkg/framework/plugins/defaultevictor/types.go` as in https://github.com/kubernetes-sigs/descheduler/pull/900/files#diff-e8b3f0a4b227d715ae7f1a0aa6a1a29ddd3fcebfad204e89a14e59f6bbf88cec. (CC @damemi)